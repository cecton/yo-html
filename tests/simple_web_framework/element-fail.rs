pub mod basic_web_framework;

use basic_web_framework::prelude::*;

fn unclosed_fragment() {
    let _ = html! { <><span /> };
}

fn unclosed_element() {
    let _ = html! { <div> };
}

fn invalid_token() {
    let _ = html! { / };
}

fn invalid_token_in_element() {
    let _ = html! { <div /!> };
}

fn main() {}
