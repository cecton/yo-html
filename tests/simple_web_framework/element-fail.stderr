error: could not find matching close fragment
 --> tests/simple_web_framework/element-fail.rs:6:21
  |
6 |     let _ = html! { <><span /> };
  |                     ^

error: could not find matching close tag
  --> tests/simple_web_framework/element-fail.rs:10:22
   |
10 |     let _ = html! { <div> };
   |                      ^^^

error: could not parse element
  --> tests/simple_web_framework/element-fail.rs:14:21
   |
14 |     let _ = html! { / };
   |                     ^

error: expected `>`
  --> tests/simple_web_framework/element-fail.rs:18:27
   |
18 |     let _ = html! { <div /!> };
   |                           ^
